src/FSharp.Control.TaskSeq.Test/TaskSeq.Do.Tests.fs

```diff
@@ -22,15 +22,15 @@ let ``CE taskSeq: use 'do!' with a task<unit>`` () =
     |> Task.map (fun _ -> value |> should equal 1)
 
 [<Fact>]
-let ``CE taskSeq: use 'do!' with a valuetask<unit>`` () =
+let ``CE taskSeq: use 'do!' with a valueTask<unit>`` () =
```

Comment:
`valueTask` is not a thing. Either we use `ValueTask`, or the lowercase for readability. We can use the proper type name.
```suggestion
let ``CE taskSeq: use 'do!' with a ValueTask<unit>`` () =
```